### ISSUE TITLE
Interface not correctly formatting

### ISSUE THREAD

**Issue by XvSkItZvX** [TehNut-Mods/HWYLA#36]
Because this has turned out to be a rather long submission I would like to take a moment to express my appreciation for your work, which is absolutely priceless, thanks in advance. If there is anything else I can do to help sort out this problem feel free to contact me.

#### Issue Description:
Interface doesn't seem to line up correctly. I'm able to move the blue box around and place it anywhere I want though options as expected. Unfortunately the text stays stacked up in the top left hand corner of my screen.

![2017-01-13_01 38 50](https://cloud.githubusercontent.com/assets/9143369/21921932/1fffc820-d931-11e6-8812-d26aa8e4106c.png)

#### Steps to reproduce:
This is constant, no actions taken.

#### Affected Versions (Do *not* use "latest"):
- Hwyla: 

Hwyla-1.8.1-B13_1.10.2.jar - Downloaded from the [Hwyla Curse Page](https://minecraft.curseforge.com/projects/hwyla) 

Waila-1.8.0-B12_1.10.2.jar - Downloaded from the [Hwyla Curse Page](https://minecraft.curseforge.com/projects/hwyla) 

Waila-1.7.0-B3_1.9.4.jar - Downloaded from the [Waila Curse Page](https://minecraft.curseforge.com/projects/waila)

- Minecraft: 1.10.2
- Forge: 1.10.2-12.18.3.2185 

#### Possible Conflicting Mods (Provide their version, as well)
Below is a list produced by ModLister then parsed into plain text.

```json
[
  {
    "n": "Advanced World Selection",
    "v": "1.11-01",
    "f": "WorldSelectionAdvanced-1.11-01.jar",
    "a": "Sedridor",
    "u": "www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2664403",
    "m": "74924fa7b987000b25948d95f6d09c15"
  },
  {
    "n": "AmbientSounds",
    "v": "1.2.0",
    "f": "AmbientSounds v1.2.1 mc1.10.2.zip",
    "a": "CreativeMD",
    "u": "http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2526167",
    "m": "6b4c6620cfc1817aa5c977db54f7b897"
  },
  {
    "n": "Better Combat",
    "v": "1.4.0",
    "f": "[MC 1.10.X] BetterCombatMod 1.4.jar",
    "a": "Ironfists",
    "u": "http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2676282-1-9-x-better-combat-mod-feat-dual-wielding",
    "m": "ba9da4d4bd36be1cdc72e4f75695d045"
  },
  {
    "n": "Better Foliage",
    "v": "2.1.3",
    "f": "BetterFoliage-MC1.10.2-2.1.3.jar",
    "a": "octarine-noise (code), Meringue (textures)",
    "u": "",
    "m": "c6d1a5c94149c659d6d84a9ed4af393e"
  },
  {
    "n": "Better Sprinting",
    "v": "1.1.7",
    "f": "BetterSprinting  MC-1.10.2  v1.1.7.jar",
    "a": "chylex",
    "u": "https://bsprint.chylex.com",
    "m": "5380e412874b591076c4b25ee833d84c"
  },
  {
    "n": "Block Drops",
    "v": "1.1.0",
    "f": "BlockDrops-1.10.2-1.1.0.jar",
    "a": "MrRiegel",
    "u": "",
    "m": "e3109aefe31c1b443b8aafa846969336"
  },
  {
    "n": "Chunk Edge Indicator",
    "v": "1.3",
    "f": "1.10-chunkedgeindicator-1.3.jar",
    "a": "",
    "u": "",
    "m": "7076bea1a2f7467d691eead1827f98c2"
  },
  {
    "n": "Client Tweaks",
    "v": "1.1.16",
    "f": "ClientTweaks_1.10.2-1.1.16.jar",
    "a": "BlayTheNinth",
    "u": "http://balyware.com",
    "m": "71d7ae2e55e9e697247627ca77ef668c"
  },
  {
    "n": "Controlling",
    "v": "1.0.3",
    "f": "Controlling-1.0.3.jar",
    "a": "Jaredlll08",
    "u": "",
    "m": "86b4dcef99ebb09ce5078ec928197bb9"
  },
  {
    "n": "Crafting Tweaks",
    "v": "6.1.16",
    "f": "CraftingTweaks_1.10.2-6.1.16.jar",
    "a": "BlayTheNinth",
    "u": "http://balyware.com",
    "m": "13486f1cc663e47c9e040bd715401111"
  },
  {
    "n": "Death Counter",
    "v": "1.1.0",
    "f": "DeathCounter-1.10.2-5.jar",
    "a": "Jiraiyah",
    "u": "http://minecraft.curseforge.com/projects/deathcounter",
    "m": "1ad5332d7c8a66ba0cc7c9d8cbd3aefc"
  },
  {
    "n": "Death Quotes",
    "v": "1.0.1",
    "f": "DeathQuotes-1.0.1-mc1.9to1.10.2-forge.jar",
    "a": "andrenoel",
    "u": "https://minecraft.curseforge.com/members/andrenoel1/projects",
    "m": "929d172a2b6bfb1d374672a2f73f910d"
  },
  {
    "n": "Dynamic Surroundings",
    "v": "3.2.1.1",
    "f": "DynamicSurroundings-1.10.2-3.2.1.1.jar",
    "a": "OreCruncher, Abastro",
    "u": "https://github.com/OreCruncher/DynamicSurroundings/wiki",
    "m": "5acf43cf9d6b5110b00df1732735ea33"
  },
  {
    "n": "Enhanced Visuals",
    "v": "1.0",
    "f": "EnhancedVisuals v1.0.7 mc1.9-1.11.jar",
    "a": "",
    "u": "",
    "m": "3a9263639ec082f18d22d0b07db881e5"
  },
  {
    "n": "EXCore",
    "v": "1.6.2",
    "f": "Excore-1.6.2-1.10.2.jar",
    "a": "Elix_x",
    "u": "",
    "m": "5d045b92565e6902d754675373617c5f"
  },
  {
    "n": "Farseek",
    "v": "2.0",
    "f": "Farseek-1.9.4-2.0.jar",
    "a": "delvr",
    "u": "https://minecraft.curseforge.com/projects/farseek",
    "m": "ff94abad744b903346da2515ed233060"
  },
  {
    "n": "Fast Leave Decay",
    "v": "1.2.3",
    "f": "FastLeaveDecay-MC1.10.2-1.2.3.jar",
    "a": "lumien",
    "u": "",
    "m": "816301259f069c9c39d6fd324be65991"
  },
  {
    "n": "Forever Enough Items",
    "v": "1.0.17.2",
    "f": "ForeverEnoughItems-1.0.17.2-1.10.2.jar",
    "a": "Elix_x",
    "u": "",
    "m": "5ea05c5713a01d0e851a496f77f87c7c"
  },
  {
    "n": "Forge Mod Loader",
    "v": "8.0.99.99",
    "f": "forge-1.10.2-12.18.3.2185-universal.jar",
    "a": "cpw, LexManos, Player",
    "u": "https://github.com/MinecraftForge/FML/wiki",
    "m": ""
  },
  {
    "n": "Fullscreen Windowed",
    "v": "1.3.0",
    "f": "FullscreenWindowed-1.10.2-1.3.0b.jar",
    "a": "Hancin",
    "u": "https://github.com/hancin/Fullscreen-Windowed-Minecraft",
    "m": "d6223b734935e419e6403f602198ab61"
  },
  {
    "n": "iChunUtil",
    "v": "6.1.0",
    "f": "iChunUtil-1.10.2-6.1.0.jar",
    "a": "iChun",
    "u": "http://ichun.me/mods/ichunutil/",
    "m": "d2f37c5faae9028332b58b3d75ce5d35"
  },
  {
    "n": "JourneyMap",
    "v": "1.10.2-5.4.2",
    "f": "journeymap-1.10.2-5.4.2.jar",
    "a": "techbrew, mysticdrew",
    "u": "http://journeymap.info",
    "m": "a8f2784b9e22e5ad4bf5eb2df618a555"
  },
  {
    "n": "Just Enough Items",
    "v": "3.14.2.401",
    "f": "jei_1.10.2-3.14.2.401.jar",
    "a": "mezz",
    "u": "https://github.com/mezz/JustEnoughItems",
    "m": "193fc382e6e4f68c7253dcc69d2e4062"
  },
  {
    "n": "Just Enough Resources",
    "v": "0.5.8.98",
    "f": "JustEnoughResources-1.10.2-0.5.8.98.jar",
    "a": "way2muchnoise",
    "u": "",
    "m": "04f218c0eee3a05870a994816a86e0a5"
  },
  {
    "n": "JustEnoughTooltips",
    "v": "2.1.39",
    "f": "JustEnoughTooltips-1.10.2-2.1.39.jar",
    "a": "superckl",
    "u": "http://minecraft.curseforge.com/projects/justenoughtooltips",
    "m": "61c311cdf69df4397a0f1bd080f5b8d3"
  },
  {
    "n": "Light Level Overlay Reloaded",
    "v": "1.1.1-mc[1.9-1.11]",
    "f": "LLOverlayReloaded-1.1.1-mc[1.9-1.11].jar",
    "a": "oldjunyi",
    "u": "www.mmyzd.com",
    "m": "a618267197c016ad8897b917b96b6a1e"
  },
  {
    "n": "Memory Bar",
    "v": "1.0.4",
    "f": "memorybar-1.10.2_1.0.4.jar",
    "a": "Henry Loenwind",
    "u": "",
    "m": "e60db045c7175e267818ef61b4e87a3d"
  },
  {
    "n": "Minecraft Coder Pack",
    "v": "9.19",
    "f": "minecraft.jar",
    "a": "",
    "u": "http://www.modcoderpack.com/website/",
    "m": ""
  },
  {
    "n": "Minecraft Forge",
    "v": "12.18.3.2185",
    "f": "forge-1.10.2-12.18.3.2185-universal.jar",
    "a": "LexManos, cpw, fry",
    "u": "http://minecraftforge.net",
    "m": ""
  },
  {
    "n": "Mo\u0027Bends",
    "v": "0.22.5",
    "f": "mobends-0.22.5_for_MC-1.10.2.jar",
    "a": "GoblinBob",
    "u": "",
    "m": "9bfd658c9aa7ad77a68554a18873399f"
  },
  {
    "n": "MobAmputation",
    "v": "6.0.0",
    "f": "MobAmputation-1.10.2-6.0.0.jar",
    "a": "iChun",
    "u": "http://ichun.me/mods/mob-amputation/",
    "m": "2ccff1f556f9a40b97e2478c58398a18"
  },
  {
    "n": "ModLister",
    "v": "1.10.2-2.1.8",
    "f": "ModLister-1.10.2-2.1.8.jar",
    "a": "iChun, Quetzi",
    "u": "http://www.modlister.com/",
    "m": "f43e98631740ee6e5de5ecacac8c4c2f"
  },
  {
    "n": "Mouse Tweaks",
    "v": "2.6.4",
    "f": "MouseTweaks-2.6.4-mc1.10.2.jar",
    "a": "YaLTeR",
    "u": "http://www.minecraftforum.net/topic/1518185-",
    "m": "6c23b8f7a876336dbab417721ee44137"
  },
  {
    "n": "Placeable Items Mod",
    "v": "3.0.3.1",
    "f": "placeableitems-3.0.3.1.jar",
    "a": "Ferdz, MasterianoX",
    "u": "http://minecraft.curseforge.com/mc-mods/227951-placeableitems",
    "m": "ad6359798d199b7c8d8a9096ae6756d1"
  },
  {
    "n": "Placement Preview",
    "v": "0.3.0",
    "f": "placementpreview-1.10.2-0.3.0.jar",
    "a": "masa",
    "u": "http://minecraft.curseforge.com/projects/placement-preview",
    "m": "8f522d3f2cab1d5abc57c8c4234f3026"
  },
  {
    "n": "Quick Hotbar",
    "v": "1.04",
    "f": "quickhotbar_1102_1.04.jar",
    "a": "Kulttuuri",
    "u": "www.quickhotbar.com",
    "m": "5a6e6ab08b5ec8d6d29f9c29c8d12bcb"
  },
  {
    "n": "Streams",
    "v": "0.3",
    "f": "Streams-1.9.4-0.3.jar",
    "a": "delvr",
    "u": "http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2346379-streams-real-flowing-rivers",
    "m": "cdc7c2a5ae1092ed0c4eeb7b906479dd"
  }
]
```

**Comment by TehNut** [comment]
Does this happen with just Hwyla?

**Comment by XvSkItZvX** [comment]
Hwyla is the only mod that appears to be having that issue, However installing Hwyla without any of the other mods in the pack it works as expected. Let me see if I can pick out which mod it is...

**Comment by XvSkItZvX** [comment]
I started with a new instance with Hwyla and began installing mods till I got the problem to present again. Seems it is an issued with [Enhanced Visuals](https://minecraft.curseforge.com/projects/enhancedvisuals)

**Comment by TehNut** [comment]
I'm going to close this as it's not an issue on my end. Sorry that he is taking so long to respond.

**Comment by XvSkItZvX** [comment]
No worries, I noticed that it also affected another mod "Quick Hotbar". I'm going to guess that he put a layer over the Interface that didn't like to play well with other mods....thanks for your time though

**Comment by CreativeMD** [comment]
Yep, it's an issue caused EnhancedVisuals, but i got now idea why, maybe the shader is doing some strange stuff ... i will try to find it out.

**Comment by CreativeMD** [comment]
It's fixed in EnhancedVisuals v1.0.9
